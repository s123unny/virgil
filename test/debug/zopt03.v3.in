b ../core/zopt03.v3 6
b ../core/zopt03.v3 28
b ../core/zopt03.v3 34
run
bt
info l
c
bt
info l
c
bt
info l
c
q
