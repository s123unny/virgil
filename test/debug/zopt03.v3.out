breakpoint|../core/zopt03.v3 6|0
breakpoint|../core/zopt03.v3 28|1
breakpoint|../core/zopt03.v3 34|2
stopOnBreakpoint
bt|getBool|../core/zopt03.v3|28
bt|Compiler.new|../core/zopt03.v3|11
bt|main|../core/zopt03.v3|32
stopOnBreakpoint
bt|SsaBlock.end|../core/zopt03.v3|6
bt|SsaCfOptimizer.optBlock|../core/zopt03.v3|23
bt|main|../core/zopt03.v3|33
variable|0|this|#1:SsaBlock|SsaBlock|true
stopOnBreakpoint
bt|main|../core/zopt03.v3|34
variable|0|a|0|int|false
variable|1|block|#1:SsaBlock|SsaBlock|true
variable|2|context|#4:SsaContext|SsaContext|true
end
result|0
